core/utils/poly-look/src/visualisations/charts/bar-charts/mirroredBarChart/mirroredBarChart.js

```diff
@@ -4,6 +4,8 @@ import { Chart } from "../../chart";
 
 import "./mirroredBarChart.css";
 
+export const type = "mirrored-bar-chart";
```

Comment:
But why do you need this? Have you considered using simply `class.constructor.name` or `instance.constructor.name`?

Reply:
This is at the same time the class of the html div the chart is located in

Reply:
So it could be a transformation from the class name... It's a great idea to have a single source of truth for this, but wouldn't it be better to link it to the class itself so that the single source of truth is the class name?

Reply:
I don't know why but it felt better calling the chart from polyChart as "vertical-bar-chart" instead of VerticalBarChart. But I can change this for sure

Reply:
But it does make sense, I'll give it a try. May look a bit odd from the CSS side of things

Reply:
You can probably just substitute a capital letter surrounded by other letters by the lower-case version preceded by a -

Reply:
Should be taken care of now